.gitignore

```diff
@@ -0,0 +1,2 @@
+*.code-workspace
+db-data/
```

Comment:
What is db-data?

Reply:
It is mounted phisical volume for Postgres database for docker's container

Reply:
Do we need it now?